webap/static/seechange.js

```diff
@@ -113,26 +124,34 @@ seechange.ExposureList.prototype.render_page = function()
 
     rkWebUtil.wipeDiv( this.parentdiv );
 
-    if ( this.div != null ) {
-        this.parentdiv.appendChild( this.div );
+    if ( this.masterdiv != null ) {
+        this.parentdiv.appendChild( this.masterdiv );
         return
     }
 
-    this.div = rkWebUtil.elemaker( "div", this.parentdiv );
+    this.masterdiv = rkWebUtil.elemaker( "div", this.parentdiv );
+
+    this.tabbed = new rkWebUtil.Tabbed( this.masterdiv );
+    this.listdiv = rkWebUtil.elemaker( "div", null );
+    this.tabbed.addTab( "exposurelist", "Exposure List", this.listdiv, true );
+    this.exposurediv = rkWebUtil.elemaker( "div", null );
+    this.tabbed.addTab( "exposuredetail", "Exposure Details", this.exposurediv, false );
+    rkWebUtil.elemaker( "p", this.exposurediv,
+                        { "text": 'No exposure listed; click on an exposure in the "Exposure List" tab.' } );
 
     var table, th, tr, td;
 
-    let p = rkWebUtil.elemaker( "p", this.div );
-    rkWebUtil.elemaker( "span", p, { "text": "[Back to exposure search]",
-                                     "classes": [ "link" ],
-                                     "click": () => { self.context.render_page() } } );
-    p.appendChild( document.createTextNode( "  —  " ) );
-    rkWebUtil.elemaker( "span", p, { "text": "[Refresh]",
-                                     "classes": [ "link" ],
-                                     "click": () => { rkWebUtil.wipeDiv( self.div );
-                                                      self.context.show_exposures(); } } );
-
-    let h2 = rkWebUtil.elemaker( "h2", this.div, { "text": "Exposures" } );
+    // let p = rkWebUtil.elemaker( "p", this.listdiv );
+    // rkWebUtil.elemaker( "span", p, { "text": "[Back to exposure search]",
+    //                                  "classes": [ "link" ],
+    //                                  "click": () => { self.context.render_page() } } );
+    // p.appendChild( document.createTextNode( "  —  " ) );
+    // rkWebUtil.elemaker( "span", p, { "text": "[Refresh]",
+    //                                  "classes": [ "link" ],
+    //                                  "click": () => { rkWebUtil.wipeDiv( self.div );
+    //                                                   self.context.show_exposures(); } } );
+
```

Comment:
do you want this commented code to survive or get deleted?

Reply:
Survive for now.,